clients/web/src/templates/body/assetstores.pug

```diff
@@ -62,10 +62,13 @@
               span.g-access-key-id  #{assetstore.get('accessKeyId')}
             div
               b Secret access key:
-              span.g-secret-key  #{assetstore.get('secret')}
+              span.g-secret-key  #{assetstore.get('secret') && '[hidden]'}
```

Comment:
This line is an unrelated change -- I just figured the secret key should be redacted from the list display page.